### PR TITLE
PAINTROID-264 Refactor ShapeTool, ImportTool to Kotlin

### PR DESCRIPTION
**Pull request by CodeChamp-SS**
Files that were refactored:

1. ShapeTool
2. ImportTool
3. ShapeToolOptionsView

https://jira.catrob.at/browse/PAINTROID-264

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/options/ShapeToolOptionsView.kt [hunk 1]
```diff
@@ -16,26 +16,19 @@
...
-		void setOutlineWidth(int outlineWidth);
-	}
+package org.catrobat.paintroid.tools.options
+
+import org.catrobat.paintroid.tools.drawable.DrawableShape
+import org.catrobat.paintroid.tools.drawable.DrawableStyle
+
+interface ShapeToolOptionsView {
+    fun setShapeActivated(shape: DrawableShape)
+    fun setDrawTypeActivated(drawType: DrawableStyle)
```
Comment (geolas94) [inline Paintroid/src/main/java/org/catrobat/paintroid/tools/options/ShapeToolOptionsView.kt:26]: functions in interface classes should have a empty line between them

**Review by geolas94 — CHANGES_REQUESTED** [review]

**Review by geolas94 — APPROVED** [review]
lgtm

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/ShapeTool.kt [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+            putInt(BUNDLE_OUTLINE_WIDTH, shapeOutlineWidth)
+        }
+    }
+
+    override fun onRestoreInstanceState(bundle: Bundle?) {
+        super.onRestoreInstanceState(bundle)
+        val newBaseShape = bundle?.getSerializable(BUNDLE_BASE_SHAPE) as DrawableShape?
+        val newShapeDrawType = bundle?.getSerializable(BUNDLE_SHAPE_DRAW_TYPE) as DrawableStyle?
+        val newShapeOutlineWidth = bundle?.getInt(BUNDLE_OUTLINE_WIDTH)
+        if (newBaseShape != null && newShapeDrawType != null && newShapeOutlineWidth != null && (baseShape != newBaseShape || shapeDrawType != newShapeDrawType)) {
```
Comment (gPathpp) [inline Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/ShapeTool.kt:115]: Please, use `?: return` instead of ` != null`.

**Review by gPathpp — CHANGES_REQUESTED** [review]

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/ShapeTool.kt (hunk 2 above)
Comment (CodeChamp-SS) [reply Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/ShapeTool.kt]: done :)

**Review by gPathpp — APPROVED** [review]
LGTM

**Pull request merged**